README.md

```diff
@@ -233,6 +233,12 @@ Run integration tests, which connects to real Cloud Spanner database.
 $ PROJECT=${PROJECT_ID} INSTANCE=${INSTANCE_ID} DATABASE=${DATABASE_ID} CREDENTIAL=${CREDENTIAL} make test
 ```
 
+Run integration tests in [CircleCI Local CLI](https://circleci.com/docs/2.0/local-cli/), which connects to [Cloud Spanner Emulator}(https://cloud.google.com/spanner/docs/emulator?hl=en)
```

Comment:
Typo: s/`}`/`]`/

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/commit/02560d5bd349e86ee98322e01e5dc29614bb52c4